src/components/global/footer.tsx

```diff
@@ -1,9 +1,223 @@
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    justify-content: center;
+  }
+`;
+
+const StyledSlider = styled.div`
+  height: 155px;
+  margin: auto;
+  width: 100%;
+  display: grid;
+  place-items: center;
+  overflow: hidden;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    height: 120px;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    height: 70px;
+  }
+`;
+
+const StyledSlideTrack = styled.div`
+  display: flex;
+  width: calc(1725px * 4); // total width of all slides
+  animation: scroll 8s linear infinite;
+
+  @keyframes scroll {
+    0% {
+      transform: translateX(0);
+    }
+    100% {
+      transform: translateX(-1725px);
+    }
+  }
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    width: calc(1200px * 4);
+    animation: scroll 6s linear infinite;
+
+    @keyframes scroll {
+      0% {
+        transform: translateX(0);
+      }
+      100% {
+        transform: translateX(-1200px);
+      }
+    }
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    width: calc(800px * 4);
+    animation: scroll 4s linear infinite;
+
+    @keyframes scroll {
+      0% {
+        transform: translateX(0);
+      }
+      100% {
+        transform: translateX(-600px);
+      }
+    }
+  }
+`;
+
+const StyledSlide = styled.div`
+  width: 1725px;
+  display: flex;
+  align-items: start;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    width: 1000px;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    width: 600px;
+  }
+`;
+
+const StyledSlideText = styled.span`
+  font-weight: 600;
+  font-size: 344px;
+  letter-spacing: 0.6rem;
+  width: 100%;
+  line-height: 72px;
+  text-shadow: 9px 0px #39979C;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    font-size: 185px;
+    line-height: 40px;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    font-size: 103px;
+    line-height: 21.6px;
+    text-shadow: none;
+  }
+`;
 
 export default function Footer() {
   return (
     <footer>
-      <h2>Footer will go here</h2>
+      <StyledWrapper>
+        <StyledContent>
+          <StyledParagraphWrapper>
+            <StyledTextSpacer>{"Interested?"}</StyledTextSpacer>
+            <StyledPillWrapper>
+              <Pill pillText={"Interested?"} />
+            </StyledPillWrapper>
+            <StyledParagraphText>
+              Get in contact, have a chat with Eoan{<StyledImage src={eoanPicture} alt="Eoan" />} or chat
+              {<Link href="/contact">
+                <StyledIconButton>
+                  <Image src={chatIcon} alt="Chat" />
+                </StyledIconButton>
+              </Link>}
+              with us
+            </StyledParagraphText>
+          </StyledParagraphWrapper>
+        </StyledContent>
+      </StyledWrapper>
+      <StyledLinkWrapper>
+        <Link href="#">Legal Information</Link>
+        <Link href="#">@WebSpace 2021</Link>
+      </StyledLinkWrapper>
+      <StyledSlider>
```

Comment:
This is really nice! Just try to make it a little lower e.g. look at the letter "E" in the Figma